File: Android/app/src/test/java/com/ReceivableServices/AccountReceivable.java
package com.ReceivableServices;

import com.DataManipulation.BaseMapperInterfaces.IBaseMapper;
import com.Domain.DomainObject;
import com.IdentityFieldAnnotation;

/**
 * Created by aiko on 4/8/17.
 */

public class AccountReceivable extends DomainObject {

    @IdentityFieldAnnotation
    public String Number;

    public String CustomerNumber ;

    public String Description;

    public AccountReceivable(String customerNumber, String number, IBaseMapper mapper)
    {
        super(mapper);

        CustomerNumber = customerNumber;
        Number = number;
    }
}

File: DotNet/CSharp/Framework.Tests/ReceivableServices/AccountReceivable.cs
using Framework.Data_Manipulation;
using Framework.Domain;

namespace Framework.Tests.ReceivableServices
{
    public class AccountReceivable: DomainObject
    {
        public AccountReceivable(IBaseMapper mapper)
            : base(mapper, null)
        {
        }

        [IdentityField]
        public string Number { get; set; }

        public string CustomerNumber { get; set; }

        public string Description { get; set; }
    }
}
